mobile/fixed_words.asm

```diff
@@ -1762,9 +1762,9 @@ Function11cb66:
 	ld a, [hl]
 	and a
 	jr nz, .asm_11cbd4
-	ld a, BANK(s4_a007)
+	ld a, BANK(sEZChatMessages)
 	call OpenSRAM
-	ld hl, s4_a007
+	ld hl, sEZChatMessages
```

Comment:
Line 1779 below this is `ld c, 12` and should be `ld c, EASY_CHAT_MESSAGE_LENGTH`.